spec/features/create_anschutz_work_spec.rb

```diff
@@ -3,7 +3,7 @@
 include Warden::Test::Helpers
 
 # NOTE: If you generated more than one work, you have to set "js: true"
-RSpec.feature "Create a AnschutzWork" do
+RSpec.feature "Create a AnschutzWork", customer: true do
```

Comment:
Is there a list of the keys that are available? There is a `slow: true` above and `customer: true` here. Whats the difference between them? Could we have just one so that we don't add extra complexity?

Reply:
There is a difference, but perhaps we could find a different name for the `:customer` tag which makes it more obvious.  The `:customer` tag is meant to apply specifically to feature tests which are all named `create_customer_name_work_type`.  So the generic tests, which don't have usage specific to a particular customer are still included in the test suite.  Perhaps it would be better if I added a tag per customer, so if you were altering things for a particular customer you could easily run those tests from the command line?  Slow tests are ones that will always be slow, that we need to run but you just don't want to run every time unless you are working on that part of the app.  You might run the full suite once including these tests before running CircleCI (which will be even slower).

Reply:
I think that might be overkill. We can already run a specific test or series of tests by specifying the specs file location and line number. 

I like the `slow: true`, as it could be applied to anything else, even the custom ones.

Reply:
I've checked this with @BertZZ and he thought customer name would be a good way to go.  It makes the tag clear.  Also I doubt you'd ever want to run all customers at once (outside of CircleCI) and you can still pick several to run at once.

Reply:
I think the issue is that we have two tags to do the same thing. The customer specs are the slow specs, so tagging them differently will only lead to the tags used interchangably.

Reply:
Now only using `:slow` tag.